torch/_prims/nvfuser_prims.py

```diff
@@ -334,6 +349,60 @@ def _amin_nvfuser(
 _nvfuser_impls["var_mean"] = _var_mean_nvfuser
 _nvfuser_impls["amax"] = _amax_nvfuser
 _nvfuser_impls["amin"] = _amin_nvfuser
+_nvfuser_impls["full"] = _full_nvfuser
+_nvfuser_impls["new_zeros"] = _new_zeros_nvfuser
+
+
+def register_new_zeros():
```

Comment:
There's `torch._refs.new_zeros` implemented in terms of `torch.full`, but `full` is implemented in terms of empty + fill, so we need `nvprims.full` here.